types.go

```diff
@@ -16,6 +16,7 @@ const (
 	// Variable length character data.
 	VarChar = "varchar"
 
+	ObjectId = "ObjectId"
```

Comment:
Please add a godoc comment to this. Otherwise, this is good to merge! Thanks for your contribution.